httpretty/core.py

```diff
@@ -508,8 +521,17 @@ def __getattr__(self, name):
             return getattr(self.truesock, name)
 
 
-def fake_wrap_socket(s, *args, **kw):
-    return s
+def fake_wrap_socket(orig_wrap_socket_fn, _socket, *args, **kw):
```

Comment:
I seem to run into a problem with this with requests/urllib3, because it calls `ssl_wrap_socket` like this:

        self.sock = ssl_wrap_socket(
            sock=conn,
            keyfile=self.key_file,
            certfile=self.cert_file,
            ssl_context=self.ssl_context,
        )

note it names the socket arg, as `sock` (which is what the original `ssl_wrap_socket` function, and python's `ssl.wrap_socket`, both call their socket argument). But because it doesn't have the same name in this fake function, I get an error that `fake_wrap_socket` didn't get enough arguments (because that `sock=` is just treated as part of `**kw`).

To fix it, just change `_socket` in this function's signature (and all the code, obviously) to `sock`.